apis/nucleus/src/index.js

```diff
@@ -419,6 +419,7 @@ function nuked(configuration = {}) {
            * @param {SearchMode=} [options.search=true] Show the search bar permanently or using the toggle button: false|true|toggle|toggleShow
            * @param {boolean=} [options.toolbar=true] Show the toolbar
            * @param {boolean=} [options.checkboxes=false] Show values as checkboxes instead of as fields
+           * @param {('left'|'center'|'right')} [options.textAlign='left'] Sets the horizontal text alignment
```

Comment:
Looking at the `spec.json` generated from the jsdoc it seems we should write it like: `left|center|right` and `options.textAlign=left` (i.e. without quotation mark `'`).